pipelines/datasets/br_inmet_bdmep/utils.py

```
@@ -232,6 +235,99 @@ def get_clima_info(file: str) -> pd.DataFrame:
     return clima
 
 
+def get_station_id_municipio(
+    sigla_uf: str, latitude: float, longitude: float
+) -> int:
+    df_municipios = bd.read_sql(
+        query=f"""
+        SELECT id_municipio, centroide
+        FROM `basedosdados.br_bd_diretorios_brasil.municipio`
+        WHERE sigla_uf = '{sigla_uf}'""",
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)